add averaged positioning funcionality doc (#230)
opengisch/QField-docs: documentation/how-to/gnss.en.md

QField. There are several options for this, one of them is [Android NTRIP Client](https://play.google.com/store/apps/details?id=com.lefebure.ntripclient).

To use this you have to [enable mock locations on your Android device](https://www.youtube.com/watch?v=v1eRHmMiRJQ).

## Averaged positioning functionality

> Note: users must have locked the coordinate cursor to their position.

Reviewer: We don't explain how to do that anywhere in the page 
```suggestion
Note: the coordinate cursor must be locked to the current location via the _lock to position_ button.
```

There is a function that allows you to digitize using averaged positions.

> The survey will start by pressing the add vertex button,

Reviewer: ```suggestion
The survey will start by pressing and holding the add vertex button,
```

which will start collecting positions.

> While collecting, an indicator will appear on top of the cursor

Reviewer: ```suggestion
While collecting, an indicator will appear on top of the coordinate cursor
```

> showing a text reflecting the current number of collected positions and, as well, a progress bar.

Reviewer: ```suggestion
showing a text reflecting the current number of collected positions. If an averaged position minimum count requirement is active, a progress bar will also be present indicating the progress towards meeting that requirement.
```

!![](../assets/images/positioning_averaged.webp)

> The progress bar reflects the advance towards meeting a user-defined minimum

Reviewer: ```suggestion
To setting to activate an average position minimum count threshold can be found in QField settings's positioning panel.
```

> collected position count. This requirement can be toggled in the settings *positioning* tab:

Reviewer: ```suggestion
When active, holding the add vertex button is not required, a short tap on the button will begin the collection of positions and automatically add the averaged position when the minimum count requirement is met.
```

## Averaged positioning functionality
:material-tablet-android:{ .device-icon } Fieldwork

!!! note
    The coordinate cursor must be locked to the current location via the [Lock to position button](../digitize/#adding-point-features)

There is a function that allows you to digitize using averaged positions.

The survey will start by pressing and holding the add vertex button, which will start collecting positions.

While collecting, an indicator will appear on top of the coordinate cursor showing a text reflecting the current number of collected positions.
If an averaged position minimum count requirement is active, a progress bar will also be present indicating the progress towards meeting that requirement.

!![](../assets/images/positioning-averaged.webp)

To setting to activate an average position minimum count threshold can be found in QField settings's *positioning* panel.
When active, holding the add vertex button is not required, a short tap on the button will begin the collection of positions and automatically add the averaged position when the minimum count requirement is met.

!![](../assets/images/positioning_averaged_set.jpg)

> In this case, when using [`@gnss_*` variables or `@position_`](../gnss/#additional-variables) it will give the mean of all collected points.

Reviewer 2: ```suggestion
When using [`@gnss_*` or `@position_` variables](../gnss/#additional-variables) on averaged positions, the variable will also represent the average over all collected samples.
```

To setting to activate an average position minimum count threshold can be found in QField settings's *positioning* panel.
When active, holding the add vertex button is not required, a short tap on the button will begin the collection of positions and automatically add the averaged position when the minimum count requirement is met.

!![](../assets/images/positioning_averaged_set.jpg)

> In this case, when using [`@gnss_*` variables](../gnss/#additional-variables) it will give the mean of all collected points.

Reviewer 2: @nirvn can you review this sentence (one thing I cannot answer is gnss or position variables)

Reviewer: That's an accurate statement

Reviewer 2: Just to be crystal clear to our doc readers, that means `gnss_` variabls are averaged, `position_` variables are not averaged?

Reviewer: Na, both would be averaged.

Reviewer 2: @lucienicolier can you add that?
